ingestors/support/convert.py

```diff
@@ -39,8 +39,8 @@ def _document_to_pdf(self, file_path, entity):
         """Converts an office document to PDF."""
         file_name = entity_filename(entity)
         mime_type = entity.first("mimeType")
+        log.debug("Converting [%s] to PDF", entity)
         for attempt in range(CONVERT_RETRIES):
```

Comment:
Minor, but this is irritatingly zero-based. I'd suggest:

```suggestion
        for attempt in range(1, CONVERT_RETRIES+1):
```

🤓

Reply:
Fair enough, i had noticed that it went 0-255 instead of 1-256. Will change.